### PR TITLE
Update Cloud documentation to reflect product changes

### LINKED ISSUE #11148: Update Cloud docs screenshots and text to remove organizations and add roles
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar request and didn't find it.
- [X] I searched the Prefect documentation for this feature.

### Prefect Version

2.x

### Describe the current behavior

Some docs Cloud section screenshots have orgs and some don't have roles.
Orgs also mentioned in the text.

### Describe the proposed behavior

Updates to current state of the world.

### Example Use

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by discdiver**
Closes #11148.

Screenshots could then use updated after forthcoming changes to UI. Would like to merge this first.

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [x] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: docs/cloud/index.md [hunk 1]
```diff
@@ -35,30 +35,33 @@ Prefect Cloud includes all the features in the open-source Prefect server plus t
...
     - Error summaries  &mdash; (enabled by Marvin AI) distill the error logs of `Failed` and `Crashed` flow runs into actionable information.
-    - [Push work pool](/guides/deployment/push-work-pools/) &mdash; run flows on your serverless infrastructure without running a worker.
+    - [Push work pools](/guides/deployment/push-work-pools/) &mdash; run flows on your serverless infrastructure without running a worker.
 
 ## User accounts
 
-When you sign up for Prefect Cloud, a personal account is automatically provisioned for you.
-A personal account gives you access to profile settings where you can view and administer your:
+When you sign up for Prefect Cloud, an account is automatically provisioned for you.
+An account gives you access to profile settings where you can view and administer your:
```
Comment (dylanbhughes) [inline docs/cloud/index.md:49]: I think we want to separate the idea of the Profile (i.e. User) and account here. 

Maybe something like:

When you sign up for Prefect Cloud, and account and a profile are automatically created for you.

Your profile is the place where you'll manage settings related to your user, including:
- Profile, including profile handle and image
- API keys
- Preferences, including timezone and color mode

As an account Admin, you will also have access to settings from the account settings page, such as:
- Billing
- Feature previews, if available
- Roles
- Account Membership
etc

File: docs/cloud/index.md [hunk 2]
```diff
@@ -35,30 +35,33 @@ Prefect Cloud includes all the features in the open-source Prefect server plus t
...
+- Feature previews, if available
 
-As a personal account owner, you can create a [workspace](#workspaces) and invite collaborators to your workspace.
+As an account owner, you can create a [workspace](#workspaces) and invite other individuals to your workspace.
 
-[Organizations](#organizations) in Prefect Cloud enable you to invite users to collaborate in your workspaces with the ability to set [role-based access controls (RBAC)](#roles-and-custom-permissions) for organization members.
-Organizations may also configure [service accounts](#service-accounts) with API keys for non-user access to the Prefect Cloud API.
+Upgrading from Prefect Cloud Free tier to a Pro or Enterprise tier plan enables additional functionality for adding workspaces, managing teams, and running higher volume workloads.
+
+Workspace Admins have the ability to use, [single sign-on (SSO)](#single-sign-on-(sso)), set [role-based access controls (RBAC)](#roles-and-custom-permissions), and configure [service accounts](#service-accounts).
```
Comment (dylanbhughes) [inline docs/cloud/index.md:60]: Pro tier gets Audit logs, just a lower retention

File: docs/cloud/rate-limits.md [hunk 3]
```diff
@@ -34,22 +34,22 @@ The Prefect Cloud API will return a `429` response with an appropriate `Retry-Af
...
 The Prefect Cloud API will return a `429` response if these limits are triggered.
 
-## Flow run retention 
+## Flow run retention
 
 !!! info "Prefect Cloud feature"
     The Flow Run Retention Policy setting is only applicable in Prefect Cloud.
 
-Flow runs in Prefect Cloud are retained according to the Flow Run Retention Policy setting in your personal account or organization profile. The policy setting applies to all workspaces owned by the personal account or organization respectively. 
+Flow runs in Prefect Cloud are retained according to the Flow Run Retention Policy setting in your account profile. The policy setting applies to all workspaces owned by the account.
```
Comment (dylanbhughes) [inline docs/cloud/rate-limits.md:46]: I would also try to only associate "profile" with the user here. Maybe:

```
Flow runs in Prefect Cloud are retained according to the Flow Run Retention Policy set by your account tier. The policy setting applies to all workspaces owned by the account.
```

File: docs/cloud/users/index.md [hunk 4]
```diff
@@ -7,19 +7,49 @@ search:
...
+
+[API keys](cloud/users/api-keys/) enable you to authenticate an environment to work with Prefect Cloud.
+
+## Service accounts <span class="badge pro"></span> <span class="badge enterprise"></span>
+
+[Service accounts](/cloud/users/service-accounts/) enable you to create a Prefect Cloud API key that is not associated with a user account.
+
+## Single sign-on (SSO) <span class="badge pro"></span> <span class="badge enterprise"></span>
+
+Organization and Enterprise plans offer [single sign-on (SSO)](cloud/users/sso/) integration with your team’s identity provider. Enterprise tier accounts provide additional options with [directory sync and SCIM provisioning](cloud/users/sso/#directory-sync).
```
Comment (dylanbhughes) [inline docs/cloud/users/index.md]:
```suggestion
Pro and Enterprise plans offer [single sign-on (SSO)](cloud/users/sso/) integration with your team’s identity provider. Enterprise tier accounts provide additional options with [directory sync and SCIM provisioning](cloud/users/sso/#directory-sync).
```

**Review by dylanbhughes — CHANGES_REQUESTED** [review]

File: docs/cloud/index.md (hunk 2 above)
Comment (discdiver) [reply docs/cloud/index.md:60]: Good catch.

File: docs/cloud/workspaces.md [hunk 5]
```diff
@@ -22,15 +22,14 @@ After creating your workspace, you'll be able to view flow runs, flows, deployme
...
+Select a workspace name in the navigation menu to see all workspaces you can access.
 
 ![Viewing all available workspaces in the Prefect Cloud UI.](/img/ui/all-workspaces.png)
 
 Your list of available workspaces may include:
 
-- Your own personal workspaces.
-- Workspaces owned by other users, who have invited you to their workspace as a collaborator.
-- Workspaces in an [organization](/ui/organizations/) to which you've been invited and have been given access.
+- Your own account workspace.
```
Comment (dylanbhughes) [inline docs/cloud/workspaces.md:31]:
```suggestion
- Your own account's workspace.
```

**Review by dylanbhughes — APPROVED** [review]

**Pull request merged**